test/validate/service_territory_test.py

```diff
@@ -13,7 +13,7 @@
     "df_name,expected_rows",
     [
         ("summarized_demand_ferc714", 3_195),
-        ("fipsified_respondents_ferc714", 135_537),
+        ("fipsified_respondents_ferc714", 135_627),
```

Comment:
I never did figure out why these row counts changed, but I verified that the records we're getting now are reasonable and understand why we're getting the ones we are and it doesn't break anything.